src/ScriptBuilder.Tests/ApprovalFiles/TimeoutScriptBuilderTest.BuildDropScript.approved.txt

```diff
@@ -1,17 +0,0 @@
-﻿declare @tableName nvarchar(max) = '[' + @schema + '].[' + @endpointName + 'TimeoutData]';
```

Comment:
Note that this file was a completely vestigial (yet committed) approval file. The real ones for these test cases end in `.ForScenario.{Dialect}.approved.txt`.